src/doc/trpl/documentation.md

```diff
@@ -373,6 +373,36 @@ we can add the `#[macro_use]` attribute. Second, we’ll need to add our own
 `main()` as well. Finally, a judicious use of `#` to comment out those two
 things, so they don’t show up in the output.
 
+Another case where the use of `#` is handy is when you want to ignore
+error handling. Lets say you want the following,
+
+```rust,ignore
+/// use std::io;
+/// let mut input = String::new(); 
+/// try!(io::stdin().read_line(&mut input));
+```
+
+The problem is that `try!` returns a `Result<T, E>` and test functions
+don't return anything so this will give a mismatched types error.
+
+```rust
+/// A doc test using try!
+///
+/// ```
+/// use std::io;
+/// # fn f() -> io::Result<()> {
+/// let mut input = String::new(); 
+/// try!(io::stdin().read_line(&mut input));
+/// # Ok(())
+/// # }
+/// # f();
+/// ```
+```
```

Comment:
Ah! Another issue. So, you actually _need_ something after a documentation comment for this to parse, so add something like

    # fn foo() { }

Does that make sense? That's why the build is failing. I think there's an example in this file...